src/OneExplorer/OneExplorer.ts

```diff
@@ -1026,8 +1025,65 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
   }
 
   /**
-   * Create ONE configuration file for a base model
-   * Input box is prefilled as <base model's name>.cfg
+   * Select the option for the configuration you want to create
+   * Return information about the selected option
+   * Default option is information about .cfg
+   *
+   * @param modelName A base model's name
+   * @param extName A base model's extension name
+   *
+   */
+  private async generateCfgInfo(
+    modelName: string,
+    extName: string
+  ): Promise<CfgInfo> {
+    const options = [
```

Comment:
How about adding title / placeHolder to the options? 😄 I recommend:
- title: Pick a configuration to create
- placeHolder: cfg / edgetpucfg

Reply:
I've done this job!